modules/virt-importing-vm-datavolume.adoc

```diff
@@ -129,7 +131,7 @@ import is complete.
 ====
 
 .Verification steps
-. The importer Pod downloads the image from the specified URL and stores it on the provisioned PV. View the status of the importer Pod by running the following command:
+. The importer Pod downloads the virtual machine image or container disk from the specified URL and stores it on the provisioned PV. View the status of the importer Pod by running the following command:
 +
 [source,terminal]
 ----
```

Comment:
<data-label> is slightly confusing, maybe we can use the term "datavolume-name" which is declared under dataVolumeTemplates.metadata.name in the yaml?

Reply:
Absolutely - thanks for the catch.
Updated it to:
----
$ oc describe dv <datavolume-name> <1>
----
<1> The name of the DataVolume as specified under `dataVolumeTemplates.metadata.name` in the virtual machine
configuration file. In the example configuration above, this is `fedora-dv`.